optimum/intel/openvino/trainer.py

```diff
@@ -153,8 +153,6 @@
         "{re}.*Embedding.*",
         "{re}.*add___.*",
         "{re}.*layer_norm_.*",
-        "{re}.*matmul_1",
-        "{re}.*__truediv__.*",
     ],
```

Comment:
Removed to fix 
```
nncf.errors.ValidationError: No match has been found among the model operations for the following ignored/target scope definitions: - ignored_scope: ['{re}.*__truediv__.*', '{re}.*matmul_1']
```

Reply:
not related to llama and gemma modeling patching so moving it to https://github.com/huggingface/optimum-intel/pull/724